pennylane/ops/functions/equal.py

```diff
@@ -52,7 +52,7 @@ def equal(
 
         The kwargs ``check_interface`` and ``check_trainability`` can only be set when
         comparing ``Operation`` objects. Comparisons of ``MeasurementProcess``
-        or ``Observable`` objects will use the defualt value of ``True`` for both, regardless
+        or ``Observable`` objects will use the default value of ``True`` for both, regardless
```

Comment:
This is unrelated but I saw it and felt a separate PR would be unnecessary

Reply:
Good idea!